manifests/init.pp

```diff
@@ -266,6 +274,8 @@
 #
 class pulp (
   $version                   = $pulp::params::version,
+  $crane_port                = $pulp::params::crane_port,
+  $crane_data_dir            = $pulp::params::crane_data_dir,
```

Comment:
It would mean that you have to include ```pulp::crane::params``` in ```pulp::params```. I'm not sure that's a good idea, mostly because it's static anyway.